opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/CohortManager.java

```diff
@@ -606,17 +608,29 @@ public QueryResult rank(String studyStr, Query query, String field, int numResul
     @Override
     public QueryResult<AnnotationSet> getAnnotationSet(String id, @Nullable String studyStr, String annotationSetName, String sessionId)
             throws CatalogException {
-        MyResourceId resource = commonGetAnnotationSet(id, studyStr, annotationSetName, sessionId);
-        return cohortDBAdaptor.getAnnotationSet(resource, annotationSetName,
-                StudyAclEntry.StudyPermissions.VIEW_COHORT_ANNOTATIONS.toString());
+        if (annotationSetName.isEmpty()) {
```

Comment:
In these cases we should be using StringUtils.isEmpty() from Apache commons lang3. That way, we will not only check for empty strings but also nulls